src/gui/CMakeLists.txt

```diff
@@ -1664,6 +1664,11 @@ if(MSVC)
     ${CMAKE_BINARY_DIR}/src/gui/qgis_gui_autogen/mocs_compilation.cpp
     PROPERTIES COMPILE_FLAGS "/bigobj"
   )
+elseif(MINGW)
+  set_source_files_properties(
+    ${CMAKE_BINARY_DIR}/src/gui/qgis_gui_autogen/mocs_compilation.cpp
+    PROPERTIES COMPILE_FLAGS "-Wa,-mbig-obj"
```

Comment:
Is this still required after the recent moc include changes?

Reply:
I don't know, 3.40.0 fails to build on MSYS2/UCRT64.

Reply:
> Is this still required after the recent moc include changes?

which change are you talking about?

Reply:
https://github.com/qgis/QGIS/pull/59230